dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java

```diff
@@ -349,6 +360,37 @@ public void onItemClick(Object item, View view, int position) {
         return view;
     }
 
+    public void loopViewsRecursively(View parent) {
+        if (parent == null) {
+            return;
+        }
+
+        if (parent instanceof ViewGroup) {
+            ViewGroup viewGroup = (ViewGroup) parent;
+            for (int i = viewGroup.getChildCount() - 1; i >= 0; i--) {
```

Comment:
I would use local variable for count.
<pre>int count= viewGroup.getChildCount() </pre>